src/views/ApplicationPage/components/DefaultInfo.tsx

```diff
@@ -65,23 +69,23 @@ const DefaultInfo = () => {
             </S.Title>
             <S.StyleBox>
               <h3>{SELECT_TYPE.CUT}</h3>
-              <StyleButton isSelected={false} type="일반 커트" />
+              <StyleButton isSelected={activateCheckbox('일반 커트')} type="일반 커트" />
```

Comment:
```suggestion
              <StyleButton isSelected={preference.includes('일반 커트')} type="일반 커트" />
```

함수로 분리하는 것은 상관없을 것 같습니다! 
배열에 특정 요소가 포함되어있는지 여부를 관리하기 위해 대표적인 메소드로 some()과 includes()가 있는데요!
- some() : 배열에 특정 **조건**을 충족시키는 요소가 하나라도 있으면 true
- includes() : 배열에 요소가 포함되어있다면 true 
이러한 차이가 있습니다. 
따라서 개인적인 견해로는 체크해줘야하는 조건이 다소 구체적이거나 복잡할 경우엔 some()을 쓰는 것이 적절하고, 
현재의 경우처럼 단순히 요소의 포함 여부만을 판단해야 할 때는 includes()가 더 적절하다고 생각합니다!